packages/manager/cypress/support/api/entityTransfer.ts

```diff
@@ -0,0 +1,66 @@
+import { EntityTransfer } from '@linode/api-v4/types';
+import {
+  getEntityTransfers,
+  cancelTransfer,
+  getLinode,
+} from '@linode/api-v4/lib';
+import { depaginate } from 'support/util/paginate';
+import { isTestLabel } from './common';
+
+/**
+ * Cancel all entity transfers whose labels are prefixed "cy-test-".
+ *
+ * @returns Promise that resolves when entity transfers have been cancelled or rejects on HTTP error.
+ */
+export const cancelAllTestEntityTransfers = async (): Promise<void> => {
+  const entityTransfers = await depaginate<EntityTransfer>((page: number) =>
+    getEntityTransfers({ page_size: 500, page })
+  );
```

Comment:
Potential improvement:

```suggestion
  const entityTransfers = (await depaginate<EntityTransfer>((page: number) =>
      getEntityTransfers({ page_size: 500, page })
    ))
    .filter((entityTransfer: EntityTransfer) => entityTransfer.status === 'pending');
```

We can `filter` out any entity transfers that aren't pending here, which will help limit the number of GET requests we make to fetch Linodes later on (and I think it will eliminate requests that result in 404 responses).

Reply:
Sounds good! Let me try that. Thank you.